addons/website_sale/models/product_template.py

```diff
@@ -643,13 +643,16 @@ def _get_additionnal_combination_info(self, product_or_template, quantity, uom,
                     product_or_template,
                     website=website,
                 )
-
-        combination_info.update({
-            'prevent_zero_price_sale': website.prevent_zero_price_sale and float_is_zero(
+        prevent_sale = False
+        if website.prevent_sale:
+            is_zero_price = float_is_zero(
```

Comment:
It would be better to define `is_zero_price=False` outside if so it would not cause any issue if we want to use it elsewhere (Just extra safety 😉 )